File: 1-TwoSum/1-TwoSum.java
// Last updated: 1/26/2026, 10:20:20 PM
1class Solution {
2    public int[] twoSum(int[] nums, int target) {
3        int[] answer = new int[2];
4        for (int i = 0; i < nums.length; i++){
5            for (int j = i+1; j < nums.length; j++){
6                if (nums[i] + nums[j] == target){
7                    answer[0] = i;
8                    answer[1] = j;
9                    break;
10                }
11            }
12        }
13        return answer;
14    }
15}

File: 1-TwoSum/1-TwoSum.py
# Last updated: 2/16/2026, 7:34:54 PM
1class Solution:
2    def twoSum(self, nums: List[int], target: int) -> List[int]:
3        seen = {}
4        for i in range(0, len(nums)):
5            if target-nums[i] in seen:
6                return [seen[target-nums[i]], i]
7            seen[nums[i]] = i